fetch/fetch_test.go

```diff
@@ -0,0 +1,143 @@
+package fetch_test
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/kansaslabs/baleen/fetch"
+)
+
+// Helper function for the httptest server to return RSS test data.
+func serveTestdata(t *testing.T, path string) http.HandlerFunc {
+	return func(rw http.ResponseWriter, req *http.Request) {
+		f, err := os.Open(path)
+		if err != nil {
+			t.Errorf("could not open testdata data: %s", err)
+		}
+
+		defer f.Close()
+
+		d, err := f.Stat()
+		if err != nil {
+			t.Errorf("could not stat RSS data: %s", err)
+		}
+
+		// Set Headers
+		rw.Header().Set("Content-Type", "text/xml")
+		rw.Header().Set("Content-Length", fmt.Sprintf("%d", d.Size()))
+		rw.WriteHeader(200)
+
+		if _, err = io.Copy(rw, f); err != nil {
+			t.Errorf("could copy RSS data to resp: %s", err)
```

Comment:
`t.Errorf("could copy RSS data to resp: %s", err)` => `t.Errorf("could not copy RSS data to resp: %s", err)`